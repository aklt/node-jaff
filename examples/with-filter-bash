#with bashOutput
#filter bash -e
echo foo | tr 'o' 'P'
echo bar | sed -e 's/b/barb/'
#end
#do
Result 1: {JSON.stringify(bashOutput[0])}
Result 2: {JSON.stringify(bashOutput[1])}
#end
