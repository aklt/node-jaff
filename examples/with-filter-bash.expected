Result 1: "fPP\n"
Result 2: "barbar\n"
